test_runner/regress/test_twophase.py

```diff
@@ -1,19 +1,32 @@
 import os
+from pathlib import Path
 
+from fixtures.common_types import TimelineId
 from fixtures.log_helper import log
-from fixtures.neon_fixtures import NeonEnv, fork_at_current_lsn
+from fixtures.neon_fixtures import (
+    NeonEnv,
+    PgBin,
+    fork_at_current_lsn,
+    import_timeline_from_vanilla_postgres,
+)
 
 
 #
 # Test branching, when a transaction is in prepared state
 #
-def test_twophase(neon_simple_env: NeonEnv):
-    env = neon_simple_env
-    endpoint = env.endpoints.create_start("main", config_lines=["max_prepared_transactions=5"])
+def twophase_test_on_timeline(env: NeonEnv):
+    endpoint = env.endpoints.create_start(
+        "test_twophase", config_lines=["max_prepared_transactions=5", "log_statement=all"]
```

Comment:
Is `log_statement=all` intended to be left or was only used for debugging and we are going to remove it later?